zebra-consensus/src/block/tests.rs

```diff
@@ -99,6 +100,8 @@ static INVALID_COINBASE_TRANSCRIPT: Lazy<Vec<(Arc<Block>, Result<block::Hash, Tr
     });
 
 #[tokio::test]
+// TODO: enable this test after implementing contextual verification
+#[ignore]
```

Comment:
The new state implementation only supports `CommitFinalizedBlock`, not `CommitBlock`, which does contextual verification. Since the block verifier does semantic verification and leaves contextual verification to the state implementation, all block verifier calls fail when they hit the state.